src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,16 @@
-const accessibilityQuiz = [
+const accessibilityQuiz = [{
```

Comment:
please fix this formatting.
You can run `npm run fix-style` to fix the formatting locally